packages/forma-36-react-components/package.json

```diff
@@ -170,7 +170,9 @@
     "extend": "3.0.2",
     "@types/react": "16.8.6",
     "set-value": "2.0.1",
-    "js-yaml": "3.13.1"
+    "js-yaml": "3.13.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
```

Comment:
Do we need this to be in resolutions, [when to use resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/#toc-why-would-you-want-to-do-this)?

Reply:
IMO we should avoid resolutions as much as possible, as they can make it difficult to understand the actual version of a package that's used. The version in a `package.json` might not actually be the version used. Also, as far as I remember they don't get updated by tools like Dependabot.

Reply:
That's why we have code reviews!
I think this is a leftover from some stuff that I was trying to do
I will remove it, thanks for spotting it 🙌